src/centralserver/admin-service/src/main/java/org/niis/xroad/centralserver/restapi/openapi/SecurityServersController.java

```diff
@@ -0,0 +1,178 @@
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * <p>
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ * <p>
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.niis.xroad.centralserver.restapi.openapi;
+
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.NotImplementedException;
+import org.niis.xroad.centralserver.openapi.SecurityServersApi;
+import org.niis.xroad.centralserver.openapi.model.CertificateDetails;
+import org.niis.xroad.centralserver.openapi.model.PagedSecurityServers;
+import org.niis.xroad.centralserver.openapi.model.PagingSortingParameters;
+import org.niis.xroad.centralserver.openapi.model.SecurityServer;
+import org.niis.xroad.centralserver.openapi.model.SecurityServerAddress;
+import org.niis.xroad.centralserver.restapi.converter.SecurityServerConverter;
+import org.niis.xroad.centralserver.restapi.service.SecurityServerService;
+import org.niis.xroad.restapi.config.audit.AuditDataHelper;
+import org.niis.xroad.restapi.config.audit.AuditEventMethod;
+import org.niis.xroad.restapi.config.audit.RestApiAuditEvent;
+import org.niis.xroad.restapi.openapi.BadRequestException;
+import org.niis.xroad.restapi.openapi.ControllerUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Locale;
+import java.util.Optional;
+import java.util.Set;
+
+import static org.niis.xroad.centralserver.restapi.entity.SecurityServer_.SERVER_CODE;
+
+@RestController
+@RequestMapping(ControllerUtil.API_V1_PREFIX)
+@PreAuthorize("denyAll")
+@RequiredArgsConstructor
+public class SecurityServersController implements SecurityServersApi {
```

Comment:
There's two naming conventions: Clients**Api**Controller vs SecurityServersController. proxy-ui-api uses "*ApiController". We could follow that, maybe also add it to https://confluence.niis.org/display/XRDDEV/API+backend+conventions

Reply:
Addressed in https://github.com/nordic-institute/X-Road/commit/b9be08a6ab77efc762a47b1cd095138749875bcd

Also documented in confluence